core/server/services/labs.js

```diff
@@ -24,7 +24,7 @@ module.exports.getAll = () => {
     const labs = _.cloneDeep(settingsCache.get('labs')) || {};
 
     ALPHA_FEATURES.forEach((alphaKey) => {
-        if (labs[alphaKey] && !(config.get('enableDeveloperExperiments'))) {
+        if (labs[alphaKey] && !(config.get('enableDeveloperExperiments') || process.env.NODE_ENV.match(/^testing/))) {
```

Comment:
Perhaps there's a better way of avoiding this when testing? I was confused as to why the PR using this change was failing, I hadn't realised in CI we also use `testing-mysql` as the NODE_ENV

Reply:
Going to merge with this in place as work is building up behind this PR. We can come back and change things later on if needed